src/Core/src/Platform/Android/WrapperView.cs

```diff
@@ -273,6 +273,8 @@ public override ViewStates Visibility
 					var child = GetChildAt(n);
 					child.Visibility = ViewStates.Visible;
 				}
+				
+				RequestLayout();
```

Comment:
Hmm, could we implement the entire `getVisibility()` method in Java instead?

https://github.com/dotnet/maui/blob/fa46d10aecc5a0fa852ecaeacfd9e1cfb2beb30d/src/Core/AndroidNative/maui/src/main/java/com/microsoft/maui/PlatformWrapperView.java#L10

It looks like the current C# `Visibility` method would do a lot of interop back and forth from Java, and this adds one more. I imagine that not being very fast, if you had a `{Binding}` on visibility and scrolled a `CollectionView`.